src/main/java/step3/domain/CarRacingRapScore.java

```diff
@@ -0,0 +1,23 @@
+package step3.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CarRacingRapScore {
+
+    private List<Car> cars;
+
+    public CarRacingRapScore(List<Car> cars) {
+        this.cars = cars.stream().map(Car::clone).collect(Collectors.toList());
```

Comment:
방어적 복사를 위해서 새로운 List를 만드신것이면 정말 좋네요 ! 👍 
Collections.unmodifiableList(cars); 를 사용하는것도 좋은 선택이 될것 같습니다. 
추가적으로 스트림은 가독성을 위해 기능 단위로 줄바꿈하는것을 권장드려요 ! 
```suggestion
        this.cars = cars.stream()
                .map(Car::clone)
                .collect(Collectors.toList());
```